bundle/Cache/PersistenceCachePurger.php

```diff
@@ -168,7 +168,7 @@ public function contentVersion($contentId, $versionNo)
         }
 
         $this->cache->deleteItem("ez-content-version-info-${contentId}-${versionNo}");
```

Comment:
you can also drop this one

Reply:
Ignore that

Reply:
As we want to make sure this works also for <2.2.2, instead change the lines here to for instance:
```php
// todo Once we can require 2.2.2+, simplify to just invalidate "content-{$contentId}-version-{$versionNo}"
$this->cache->deleteItem("ez-content-version-info-${contentId}-${versionNo}");
$this->cache->invalidateTags(["content-${contentId}-version-list", "content-{$contentId}-version-{$versionNo}"]);
```

Reply:
@andrerom So I don't need do anything, do I?

Reply:
Not anymore at least ;)